loggo/_loggo.py

```diff
@@ -223,6 +223,8 @@ def _decorate_all_methods(self, cls: type, just_errors: bool = False) -> type:
         members = inspect.getmembers(cls)
         members = [(k, v) for k, v in members if callable(v) and self._can_decorate(v, name=k)]
         for name, candidate in members:
+            if name not in vars(cls):
```

Comment:
wouldn't be better to put the logic that ignores parent class' methods in self._can_decorate()?